controllers/sriov-controller/Dockerfile.client

```diff
@@ -0,0 +1,27 @@
+FROM centos:latest
```

Comment:
Why CentOS? Just curious.

Reply:
Because I needed a bunch of packages for troubleshooting vfio, I did not know and did not have to to find the same for alpine.

Reply:
Note that this may make this image large.